www/css-utils/card/readme.md

```diff
@@ -9,59 +9,49 @@ To apply card css utils, we can use 'fw-card-#{$elevation}'
 ## Sample - card with interactive component 
 ```html live
   <template>
-    <div>
-      <div class="fw-card-1 fw-p-24 fw-flex fw-flex-row">
-        <div class="fw-flex-grow">
-          <div class="fw-type-h5">Arabic</div>
-          <div class="fw-type-xs">Last updated - 25 June 2020</div>
-        </div>
-        <div class="fw-flex-grow-0">
-          <fw-button color="secondary" class="fw-type-h6"> Download existing </fw-button>
-          <fw-button color="secondary" class="fw-type-h6 fw-ml-8"> Update file </fw-button>
-        </div>
-        <div class="fw-flex-grow-0">
-          <fw-button size="icon" color="text" role="button" class="fw-ml-12">
-            <fw-icon name="delete" ></fw-icon>
-          </fw-button>
-        </div>
-      </div>
+    <div class="fw-card-1 fw-p-24 fw-flex fw-flex-row">
```

Comment:
can you also change it to `section`

Reply:
If the element is more of a styling wrapper, documentation suggests to use div. So, leaving it as such.

Ref link:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section